### ISSUE TITLE
prometheus_client 0.7 version appears to break stuff

### ISSUE THREAD

**Issue by timmycarr** [sbrudenell/sb8200_exporter#1]
A bump in the prometheus_client code appears to have broken stuff. I'm taking a look in my spare time. If I set prometheus_client==0.6 things work for me.

**Comment by sbrudenell** [comment]
You're right. I didn't do the imports quite right and the update broke it
